pytest_flask/plugin.py

```diff
@@ -165,6 +165,8 @@ def pytest_addoption(parser):
                     help='use a host where to listen (default localhost).')
     group.addoption('--live-server-port', action='store', default=0, type=int,
                     help='use a fixed port for the live_server fixture.')
+    group.addoption('--live-server-scope', action='store', default='function', type=str,
```

Comment:
As I suggested earlier, let's make this an `ini` option instead. 👍

Reply:
The next release will be `1.0`, and given that @havok2063 mentioned he has been using `live_server` as "session" for a long time, how about we change the default to `session`?

Reply:
Sounds good! I wasn't sure if we wanted to support `cli` as well

Reply:
The good thing is that you can change ini options directly in the command-line: `pytest -o live_server_scope=function` for example.